.travis.yml

```diff
@@ -9,3 +9,4 @@ before_script:
   - 'yarn'
 script:
   - yarn run eslint
+  - node __tests__/locales.test.js
```

Comment:
Probably shouldn't do this here – this is going to cause a lot off issues with #407 and will end up just being moved out. Try to refactor this to the `tests` branch and make a PR there and it'll show up in #407. I can definitely help out in this area.

Reply:
@andrewda I picked out commit [66b3228](https://github.com/gitpoint/git-point/commit/66b32289515e3b543fbeb5dd5b319fea55f2bb5d) and kept this PR focus on translations.

Do you have any suggestions for where I can put `locales.test.js`? I think `__tests__` is not a good place, because it only contains `jest` testing files.